conanfile.txt

```diff
@@ -1,16 +1,16 @@
 [requires]
-CLI11/1.9.0@cliutils/stable
-librdkafka/1.5.0@ess-dmsc/stable
-gtest/1.8.1@bincrafters/stable
-fmt/6.0.0
-spdlog/1.4.2
-jsonformoderncpp/3.7.3@vthiery/stable
-streaming-data-types/3966e53@ess-dmsc/stable
-date/2.4.1@bincrafters/stable
-OpenSSL/1.1.1c@conan/stable
+cli11/1.9.1
+librdkafka/1.6.0
+gtest/1.10.0
+fmt/6.2.0
+spdlog/1.8.5
+nlohmann_json/3.9.1
+streaming-data-types/6a41aee@ess-dmsc/stable
+date/2.4.1
 
 [generators]
 cmake
+cmake_find_package
```

Comment:
This is not required when using `CONAN_PKG` namespace targets, right?

Reply:
Thank you for the review. You're right, I didn't pay attention to the fact that we were using the `CONAN_PKG` namespace.